lisp/core.slosh

```diff
@@ -94,11 +94,12 @@ Example:
 (test::assert-true (seq? '(1 2 3)))
 (test::assert-true (seq? [1 2 3]))
 (test::assert-true (seq? '()))
+(test::assert-true (seq? '()))
 (test::assert-true (seq? []))
 (test::assert-false (seq? "aaa"))
 (test::assert-false (seq? 1))
 %#
-(def seq? (fn (v) (or (pair? v)(vec? v))))
+(def seq? (fn (v) (or (nil? v)(pair? v)(vec? v))))
```

Comment:
changed function in lisp here but, should pair? recognize Nil?

Reply:
I don't think so.  I know this could go either way but especially with xar!/xdr! needing a pair not nil I think it is best for it not to.  At least for now.  We need to have a big language discussion soon to hash out how we want some of this stuff to feel.  For instance, I am not sure having Nil/EmptyList may not make sense.

Reply:
done.